profile.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Profile</title>
+    <link rel="stylesheet" href="/dist/css/styles.css">
+
+</head>
+
+<body>
+    <div class="container">
```

Comment:
This is not perfectly responsive. PLs, have a look. You can start by removing the div container.